modules/mapper-extras/src/test/java/org/opensearch/index/mapper/ScaledFloatFieldTypeTests.java

```diff
@@ -83,56 +82,6 @@ public void testTermsQuery() {
         assertEquals(LongField.newSetQuery("scaled_float", scaledValue1, scaledValue2), ft.termsQuery(Arrays.asList(value1, value2), null));
     }
 
-    public void testRangeQuery() throws IOException {
-        // make sure the accuracy loss of scaled floats only occurs at index time
-        // this test checks that searching scaled floats yields the same results as
-        // searching doubles that are rounded to the closest half float
-        ScaledFloatFieldMapper.ScaledFloatFieldType ft = new ScaledFloatFieldMapper.ScaledFloatFieldType(
```

Comment:
As comments in the test mentioned it assumed only precision loss during indexing and it compared with double when used BigDecimal. This is not the case now as we are using direct multiplication same as indexing.